Grunt release fails due to missing create-bundles script

### Expected behavior

The `grunt release` task should build the release successfully.

### Actual behavior

The `grunt release` task fails with the following error:

```
Running "exec:folder" (exec) task

Running "obfuscator" task

Running "exec:createBundles" (exec) task
>> npm
>> ERR! Darwin 17.3.0
>> npm ERR! argv
>> "/usr/local/bin/node" "/usr/local/bin/npm" "run" "create-bundles"
>> npm
>> ERR! node v6.12.3
>> npm
>> ERR! npm
>> v3.10.10
>> 
>> npm ERR! missing script: create-bundles
>> npm ERR! 
>> npm ERR! If you need help, you may report this error at:
>> npm
>> ERR!     <https://github.com/npm/npm/issues>
>> 
>> npm ERR! Please include the following file with any support request:
>> npm ERR!
>> /Users/Oliver/GitHub/LiskHQ/lisk-development/npm-debug.log
>> Exited with code: 1.
>> Error executing child process: Error: Process exited with code 1.
Warning: Task "exec:createBundles" failed. Use --force to continue.

Aborted due to warnings.
```

### Steps to reproduce

Run the `grunt release` task.

### Which version(s) does this affect? (Environment, OS, etc...)

On the `1.0.0` branch.

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -101,7 +101,7 @@ module.exports = function (grunt) {
 			},
 
 			createBundles: {
-				command: 'npm run create-bundles',
+				command: 'npm run build',
 				maxBuffer: maxBufferSize
 			},
 

--- a/test/functional/system/1.X/1.0.transfer.js
+++ b/test/functional/system/1.X/1.0.transfer.js
@@ -23,7 +23,7 @@ var sendTransactionPromise = require('../../../common/helpers/api').sendTransact
 var randomUtil = require('../../../common/utils/random');
 var errorCodes = require('../../../../helpers/apiCodes');
 
-describe('POST /api/transactions (unconfirmed type 0 on top of type 1)', function () {
+describe('POST /api/transactions (unconfirmed type 0 on top of type 1) @unstable', function () {
 
 	var transaction;
 	var badTransactions = [];

